src/ui/components/DotComponent.tsx

```diff
@@ -57,20 +54,15 @@ export const DotComponent: FunctionComponent<DotComponentProps> = (props) => {
   }
 
   return (
-    <Touchable
-      activeOpacity={1}
-      onPress={props.onPress}
-      testID="button"
-      disabled={!clickable}
-      aria-hidden={!clickable}>
-      <DotContainer>
-        <Dot
-          accessibilityLabel={clickable ? `Étape ${step} sur ${totalSteps} ${status}` : undefined}
-          size={props.isActive ? CURRENT_STEP_SIZE : DEFAULT_SIZE}
-          testID="dot-icon"
-        />
-      </DotContainer>
-    </Touchable>
+    <DotContainer>
+      <Dot
+        accessibilityLabel={
+          props.isActive ? `Étape ${step} sur ${totalSteps} ${status}` : undefined
+        }
```

Comment:
Il faut toujours garder le label, pour indiquer qu'il y a des étapes qui ont été faites ou qui sont à faire 
```suggestion
        accessibilityLabel={`Étape ${step} sur ${totalSteps} ${status}`}
```

Reply:
Est-ce qu'on a pas assez d'infos avec "Etape 2 sur 4 en cours" (2 étapes faites, 2 restantes, 4 en tout ?)
Parce que sinon dans l'inscription avec le voice over on va avoir "Etape 1 sur 4 réalisée" "Etape 2 sur 4 en cours" "Etape 3 sur 4 à faire" "Etape 4 sur 4 à faire" c'est pas un peu trop lourd ? (On y a réfléchi avec Lucas)

Reply:
Certes c'est lourd mais c'est ce qui a été demandé par Tanaguru

Reply:
Trop chiant Tanaguru mais si c'est une obligation, il faut le faire :(